### ISSUE TITLE
Cannot read properties of undefined (reading 'timeout')

### ISSUE THREAD

**Issue by ar-naturalint** [DataDog/dd-trace-js#3939]
need to check that `options` exists

```
uncaughtException: Cannot read properties of undefined (reading 'timeout')
TypeError: Cannot read properties of undefined (reading 'timeout')
    at HttpClientPlugin.error (/app/node_modules/dd-trace/packages/datadog-plugin-http/src/client.js:125:39)
    at /app/node_modules/dd-trace/packages/dd-trace/src/plugins/tracing.js:73:22
```

https://github.com/DataDog/dd-trace-js/blob/cfc243638ed8142e1fcf90f8d56ee11631c0c521/packages/datadog-plugin-http/src/client.js#L125

**Comment by resteinbock** [comment]
We cannot use https://github.com/DataDog/dd-trace-js/releases/tag/v4.23.0 because of this issue

**Comment by atif-saddique-deel** [comment]
any update on this? we are also having this from time to time.

**Comment by simon-id** [comment]
The fix has been released in versions `5.3.0` / `4.27.0` / `3.48.0` of `dd-trace` :+1: Thank you @atif-saddique-deel for the PR!
